### PR TITLE
007 role matrix

### PR DESCRIPTION
**Pull request by CKindred**

### CODE REVIEW

File: routes/roleMatrixRoutes.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+const { getBands } = require('../services/bandService');
+const { getCapabilityLeads } = require('../services/capabilityService');
+const {getRolesSortedByCapability} = require('../services/jobRoleService');
```
Comment (TheoHodges) [inline routes/roleMatrixRoutes.js:4]: Nitpick but inconsistent spacing here

File: selenium/roleMatrix.test.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    afterEach(async () => {
+        await driver.quit();
+    });
+
+    test("Header should say 'Job Role Matrix'", async () => {
+        await driver.get('http://localhost:8000/role-matrix').then(async () => {
+            let header = await driver.findElement(webdriver.By.xpath('//h1'));
+            let expected = "Job Role Matrix";
+            let headerText = await header.getText();
+            expect(headerText).toBe(expected);
```
Comment (TheoHodges) [inline selenium/roleMatrix.test.js:25]: Not sure whether act and assert should be separated by whitespace?

File: services/jobRoleService.js [hunk 3]
```diff
@@ -4,7 +4,8 @@ exports.getRolesSortedByCapability = async () => {
     let results;
     await axios.get(`http://localhost:8080/job-role/list-sorted`)
         .then(response => {
-            results = response.data; 
+            results = response.data;
+            console.log(results);
```
Comment (TheoHodges) [inline services/jobRoleService.js:8]: Remove console.log pls

File: views/roleMatrix.njk [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+                        {%endif%}
+                    {%endfor%}
+                    </td>
+            {%endfor%}
+        </tr>
+    {%endfor%}
+<table>
+
+
+
```
Comment (TheoHodges) [inline views/roleMatrix.njk:38]: How much whitespace?

**Review by michaltangri — CHANGES_REQUESTED** [review]
Your Travis build failed, because it tried to run Selenium tests with Unit tests.
Please change `"test": "NODE_ENV=test jest --forceExit",` in **package.json** to `"test": "NODE_ENV=test jest ./tests --forceExit",`

**Review by TheoHodges — CHANGES_REQUESTED** [review]
Looks fine - just minor formatting issues and a console.log that needs removing

**Review by TheoHodges — APPROVED** [review]

**Review by michaltangri — APPROVED** [review]

**Pull request merged**